README.md

```diff
@@ -174,9 +174,16 @@ let yellowBackground = TextStyle(
     .backgroundColor(.yellow)
 )
 
+let secret = TextStyle(
+    .writingDirection([
```

Comment:
I wouldn't add this option. rather natural would be much better since if the direction changes there are often issues with flipping if forced.

Reply:
natural is for paragraph styles. this are the overrides: https://developer.apple.com/documentation/foundation/nsattributedstringkey/1524497-writingdirection.
Maybe rename?

Reply:
yep, that would be betters since it leads to this misconception 👍